src/Xhr.js

```diff
@@ -0,0 +1,23 @@
+import Cookies from 'js-cookie';
+
+export default class Xhr {
+    createSequenceAsset(spineVidId, courseId, projectId) {
+        const csrftoken = Cookies.get('csrftoken');
+        fetch('/sequence/api/assets/', {
```

Comment:
Not a criticism, but hoping to maybe start a conversation.

Do we have any decent techniques/patterns for not having endpoint URL paths hard-coded in our JS?

On the Django side, we can name URLs and use `{% url ... %}` and such and that makes it really easy to change URL structures, move modules around etc. But as soon as we have JS talking to the endpoints, that seems to go out the window.

Here and there I've tried things like assigning the paths to variables from within the HTML templates, but that doesn't seem to scale beyond pretty basic use-cases and really doesn't work once we get into SPAs.